requirements/requirements_multimodal.txt

```diff
@@ -0,0 +1,20 @@
+addict
+basicsr
+clip
+diffusers>=0.19.3
+einops_exts
+# webdataset>=0.2.48 conflict with ASR models, manually pip install when using multimodal
+# flash-attn>=2.0.1
+imageio-ffmpeg
```

Comment:
What is image io ffmpeg used for? If installs it's own version of ffmpeg which can conflict with ASR required versions of ffmpeg

Reply:
Plus last commits were 4 months ago !

Reply:
Used by NERF. Asking PIC to answer.

Reply:
imageio is used to render a video from of a series of images. the code doesn't require a specific version so I'll check if we can remove the dependency from the requirements file or relax the dependency to allow any installed version